src/ui/wxWidgets/passwordsafeframe.cpp

```diff
@@ -847,6 +848,8 @@ void PasswordSafeFrame::ShowGrid(bool show)
 
 void PasswordSafeFrame::ShowTree(bool show)
 {
+  static bool initialShowTree = true;
```

Comment:
This is problematic , and explains why the TreeDisplayStatusAtOpen code below didn't work for me:
Consider what happens when the program isn't exited, but another file is opened via File->Open. In that case, changes to the preference won't be respected.
Another way needs to be found to track the initialShowTree value. Simplest is a bool data member set by the File->Open flow, and cleared here.